WalletWasabi/EventSourcing/InMemoryEventRepository.cs

```diff
@@ -208,19 +212,22 @@ private void IndexNewAggregateId(string aggregateType, string aggregateId)
 				comparisonValue: new AggregateTypeIds(tailIndex, aggregateIds)));
 		}
 
-		// Helper for parallel critical section testing in DEBUG build only.
+		// Hook for parallel critical section testing in DEBUG build only.
+		[Conditional("DEBUG")]
```

Comment:
I think kiminuo's argument for removing the conditionality still makes sense, running the tests with a release build still makes sense and performance hit is not a concern

Reply:
Yeah, I changed it to kiminuo way eventually in the next pull request